.changeset/early-ghosts-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@equinor/fusion-framework-react-module-context": minor
+"@equinor/fusion-framework-module-context": minor
+---
+
+Add FusionContextSearchError
```

Comment:
Add note about possible breaking change of `packages/modules/context/src/ContextProvider.ts`

Reply:
Updated